docker-compose.yml

```diff
@@ -29,6 +29,7 @@ services:
       - STORAGE_API_TOKEN_MASTER
       - RUNNER_COMMAND_TO_GET_HOST_IP
       - STORAGE_API_TOKEN_READ_ONLY
+      - STORAGE_API_TOKEN_FIXED_TYPE
```

Comment:
I sort of don't understand this name at all :)

Reply:
yeah that name is debilny.  how about `STORAGE_API_TOKEN_TYPED_TABLES` ?

Reply:
definitely better, or STORAGE_API_TOKEN_FEATURE_TABLES_DEFINITION ?

Reply:
surre, thanks 🍻